src/memory/AmdFlash.cc

```diff
@@ -55,6 +55,7 @@ void AmdFlash::init(const std::string& name, const DeviceConfig& config, Load lo
 	assert(std::popcount(to_underlying(chip.autoSelect.manufacturer)) & 1); // check parity
 	assert(chip.autoSelect.device[0] == 0x7E ||
 		(chip.autoSelect.device[1] == 0x00 && chip.autoSelect.device[2] == 0x00)); // single or triple
+	assert(!chip.program.fastCommand || chip.program.pageSize.exponent > 0); // fast program requires page size > 1
```

Comment:
Detail:  why `pageSize.exponent > 0` instead of `pageSize() > 1`   (or without `()` if the implicit conversion operator idea works fine). Then you don't need the comment explaining what's going on.

Reply:
Will do.